src/ophyd_async/epics/adaravis/_aravis.py

```diff
@@ -29,17 +30,23 @@ def __init__(
         controller, driver = AravisController.controller_and_drv(
             prefix + drv_suffix, gpio_number=gpio_number, name=name
         )
+        writer, fileio = writer_cls.writer_and_io(
+            prefix,
+            path_provider,
+            lambda: name,
```

Comment:
```suggestion
            lambda: self.name,
```
Or event better let `writer_cls.writer_and_io` do this as in my PR